### ISSUE TITLE
Zombiefied Piglins lose anger to turtle eggs (forever) (Nether issue)

### ISSUE THREAD

**Issue by AlienGurke** [LunaPixelStudios/Better-Minecraft#426]
### Describe the Bug

I wanted to build a Witherskelton farm in creative mode and I noticed that Zombiefied Piglins were not attracted to Turtle Eggs. So I experimented a bit and came up with these results: 
-its not a MC Vanilla issue
-happens in every world and also server (in bmc 1.18.2 forge)
-doesnt happen at the spawn in Nether (x=0 z=0)
-doesnt happen in the Overworld

Due to this bug, farms that need Turtle Eggs are not possible or do not work efficiently. 
(f.e. wither skeleton farms or zombiefied piglin farms)

>mobGriefing true
>more than enough air over the Turtle Egg

### Reproduction Steps

1. Build a nether portal near the spawn.
2. Go in the Nether and place a Turtle Egg there and a Zombiefied Piglin: You can see that they get angered at the spawn.
3. If you now do f.e."/tp 10000 100 10000" and u place again a Turtle Egg and spawn a Zombiefied Piglin, u will notice that it totally ignores the Egg.
5. You can build a portal there and try the same in the Overworld. You will be at around x=80000 and z=80000 and as u can see there is no issue in the Overworld, it only happens in the Nether.
4. If u do the same in MC Vanilla 1.18.2, you will see that there is no issue in the Nether and the Zombiefied Piglin will instantly run to the Egg and will destroy it.

### Screenshots and Videos
Better Minecraft Issue:
https://youtu.be/lXjo8XpQSZQ

how it works in Minecraft Vanilla:
https://www.youtube.com/watch?v=aPVTw1-PcUQ

### Operating System

Windows 11

### Minecraft Version

1.18

### Modloader

Forge

### Modpack Version

bmc v11.5

### Optifine

No

### Other Mods

no

**Comment by AlienGurke** [comment]
+added the (video) comparison between vanilla and better minecraft mod

**Comment by Logicalita** [comment]
This also happened to me. It does actually take place in the Nether only. I had play for several hundred mc days so when I decided to go make a gold farm I couldn’t make a decent one. I tried to travel back to spawn chunk and test the turtle eggs there. The zombified piglins trample those as intended, but when I tried to make the farm there (at spawn chunk) on y=118 & y=121, they were broken again and ignored the eggs.
